src/index.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import { Provider } from 'react-redux';
 import App from './App';
+import store from './redux/store';
+import { fetchMissions } from './redux/missions/missionsSlice';
+
+store.dispatch(fetchMissions());
 
```

Comment:
It's not mandatory to call the fetch data inside the `useEffect` hook, but it's a recommended approach because it ensures that the fetch is only called once when the component mounts, and not every time the component re-renders. so it would be better if you use the `useEffect` to call the `fetchMissions` function inside the `Navbar` file.

Reply:
Thank you @AbdusSattar-70  for your code review. I Will be working on the suggested changes ASAP 😄